<launch>
  <!-- Maryland Park -->
  <!-- <param name="ref_lat" value="38.379318"/>
  <param name="ref_lon" value="-76.805290"/> -->

  <!-- Naval Air Base -->
  <!-- <param name="ref_lat" value="38.146293"/>
  <param name="ref_lon" value="-76.429509"/>
  <param name="ref_h" value="0"/> -->

  <!-- Springville -->
  <!-- <param name="ref_lat" value="40.174571"/>
  <param name="ref_lon" value="-111.651641"/>
  <param name="ref_h" value="1402.08"/> -->

  <!-- Elberta -->
  <param name="ref_lat" value="39.982930"/>
  <param name="ref_lon" value="-111.989435"/>
  <param name="ref_h" value="1502.00"/>

  <param name="target_h" value="60" />
  <param name="SEARCH" value="2"/>
  <param name="PAYLOAD" value="1"/>
  <param name="LOITER" value="7"/>
  <param name="OBJECTIVE" value="4"/>
  <param name='DROP_LOCATION' value="[99999.,99999.,99999.]"/>
  <param name="ARMED_AND_READY" value="false"/>

  <node name="mission_planner" pkg="metis" type="main" output = "screen" />
  <!-- <node name="interop_client" pkg="interop_pkg" type="InteropClient.py" output="screen">
    <rosparam file="$(find interop_pkg)/param/client_params.yaml"/>
  </node> -->
  <node name="payload_drop" pkg="metis" type="payload_drop" output="screen" />


</launch>
